src/VisualStudio/Setup/VisualStudioSetup.csproj

```diff
@@ -198,6 +198,10 @@
       <NgenArchitecture>All</NgenArchitecture>
       <NgenPriority>3</NgenPriority>
     </ProjectReference>
+    <ProjectReference Include="..\Setup.Dependencies\VisualStudioSetup.Dependencies.csproj">
+      <Project>{1688e1e5-d510-4e06-86f3-f8db10b1393d}</Project>
+      <Name>VisualStudioSetup.Dependencies</Name>
+    </ProjectReference>
```

Comment:
Please verify we don't end up including the output of the .Dependencies project in this project, as that'd be Most Strange™. I suspect unless you override `IncludeOutputGroupsInVsix` that'll probably happen.

@jaredpar Would any of your various *Util tools squawk if a new DLL slipped into a VSIX we signed without mentioning that was expected?